### PR TITLE
Remove publish profile from logs

### PR DESCRIPTION
**Pull request by SumiranAgg**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ async function main() {
 
     // Initialize action inputs
     let endpoint: IAuthorizer = !!core.getInput('publish-profile') ? null : await AuthorizerFactory.getAuthorizer();
-    let actionParams: ActionParameters = ActionParameters.getActionParams(endpoint);
+    ActionParameters.getActionParams(endpoint);
```
Comment (asranja) [inline src/main.ts]: required ?

File: src/ActionInputValidator/Validations.ts [hunk 2]
```diff
@@ -33,9 +33,9 @@ export function validateAppDetails() {
 
     if(!!actionParams.appName || (!!actionParams.slotName && actionParams.slotName !== 'production')) {
```
Comment (asranja) [inline src/ActionInputValidator/Validations.ts]: Assuming there will not be an issue with case-sensitivity here.

File: src/ActionInputValidator/Validations.ts [hunk 3]
```diff
@@ -33,9 +33,9 @@ export function validateAppDetails() {
 
     if(!!actionParams.appName || (!!actionParams.slotName && actionParams.slotName !== 'production')) {
         let creds: ScmCredentials = PublishProfile.getPublishProfile(actionParams.publishProfileContent).creds;
-        let splitUsername: string[] = creds.username.substring(1).split("__");
-        let appNameMatch: boolean = !actionParams.appName || actionParams.appName === splitUsername[0];
-        let slotNameMatch: boolean = actionParams.slotName === 'production' || actionParams.slotName === splitUsername[1];
+        let splitUsername: string[] = creds.username.toUpperCase().substring(1).split("__");
+        let appNameMatch: boolean = !actionParams.appName || actionParams.appName.toUpperCase() === splitUsername[0];
```
Comment (asranja) [inline src/ActionInputValidator/Validations.ts]: It is not really required to specify type for all cases. Typescript will auto detect the type in most cases.

Comment (SumiranAgg) [reply src/ActionInputValidator/Validations.ts]: It does but this is best practice. typescript by definition allows you to define types for your variables. Just trying to use that capability explicitly

File: src/ActionInputValidator/Validations.ts (hunk 2 above)
Comment (SumiranAgg) [reply src/ActionInputValidator/Validations.ts]: This condition is to check default case, which we ourselves have set if user doesn't provide a value

**Pull request merged**